### PR TITLE
Updating upstart template regarding exe copy, runtime

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

File: utils/pkg/templates/upstart_travis_worker.conf [hunk 1]
```diff
@@ -34,17 +34,18 @@ script
     . /etc/default/travis-enterprise
   fi
 
-  if [ -f /etc/default/travis-worker ] ; then
-    . /etc/default/travis-worker
+  if [ -f /etc/default/$UPSTART_JOB ] ; then
```
Comment (solarce) [inline utils/pkg/templates/upstart_travis_worker.conf]: Does this come from Upstart automatically?

**Comment by henrikhodne** [comment]
:shipit: :sparkles: :heart: :tada:

File: utils/pkg/templates/upstart_travis_worker.conf (hunk 1 above)
Comment (meatballhat) [inline utils/pkg/templates/upstart_travis_worker.conf]: confirm! http://upstart.ubuntu.com/cookbook/#standard-environment-variables

**Comment by solarce** [comment]
:sheep: :one:

**Pull request merged**